gadget/install/install.go

```diff
@@ -176,12 +176,18 @@ func Run(gadgetRoot, device string, options Options) error {
 		loadChain = append(loadChain, options.KernelPath)
 	}
 
-	// TODO:UC20: get cmdline definition from bootloaders
+	// Get kernel command lines
+	cmdline, err := boot.ComposeCommandLine(options.Model)
```

Comment:
This should be ComposeCandidateCommandLine(), preferrably with a comment that we're obtaining a expected kernel command line for the system that is currently being installed.

Reply:
Updated and added comment as suggested.